hack/smoke-test.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/sh
 
+shout() {
```

Comment:
I checked out you PR and ran `shellcheck` ... here are the items it reported for this one:

```
$ shellcheck smoke-test.sh 

In smoke-test.sh line 5:
  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
       ^-- SC3037: In POSIX sh, echo flags are undefined.


In smoke-test.sh line 49:
    if [[ $i == "testing-namespace"* ]]; then
       ^-- SC3010: In POSIX sh, [[ ]] is undefined.


In smoke-test.sh line 50:
        oc delete project $i
                          ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
        oc delete project "$i"

For more information:
  https://www.shellcheck.net/wiki/SC3010 -- In POSIX sh, [[ ]] is undefined.
  https://www.shellcheck.net/wiki/SC3037 -- In POSIX sh, echo flags are undef...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

Reply:
Thanks @gabemontero for pointing it out. `shellcheck` error has been fixed now. If you could have a look 🙂